.cascade-code/Chapman.edu/test-section/tatwell/fast-omni-nav/debug_format.vtl

```diff
@@ -0,0 +1,37 @@
+##
```

Comment:
This is just a test format that can be used for page content. It demonstrates some basic debug helpers and Velocity collection datatypes.

Note `.vtl` extension. Going forward, I propose all Velocity formats in Cascade include this in their name and follow Rails snake_case file naming convention.

Reply:
👍

Reply:
The left nav formats I wrote are not under `Chapman.edu/` in `.cascade-code`, should we have an action item or something to move them? They are under that folder in Cascade obviously, I just didn't copy that part to .cascade-code which I probably should have.

Reply:
Yes, add an action item for it. It's not urgent.

It's very Java-esque to include it but that's how Cascade lists out the path so let's include it and try to be consistent.